src/cozmo/run.py

```diff
@@ -276,6 +277,8 @@ def _remove_forward(self, serial):
                     try:
                         await conn_check(proto)
                     except Exception as e:
+                        if isinstance(e, exceptions.SDKVersionMismatch):
```

Comment:
Might be better to have a separate `except exceptions.SDKVersionMismatch as e:` branch than to use `isinstance` here

Reply:
I did originally do it that way, but I felt it was duplicating more code that way. I'm not hugely torn either way - if there's a best-of-both-worlds I'd be interested?